src/Resources/translations/SonataAdminBundle.nl.xliff

```diff
@@ -472,12 +472,12 @@
             </trans-unit>
             <trans-unit id="read_more">
                 <source>read_more</source>
-                <target>read_more</target>
+                <target>Lees verder</target>
             </trans-unit>
             <trans-unit id="read_less">
                 <source>read_less</source>
-                <target>read_less</target>
+                <target>Lees minder</target>
             </trans-unit>
         </body>
     </file>
-</xliff>
+</xliff>
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.

Reply:
So expected is that there is always A newline at the end of the file?

Reply:
Yes, because you should be able to see a file as a collection of lines, and a line as a collection of characters ending with a newline, no exceptions.

Reply:
@greg0ire Maybe we could include a [`.editorconfig` ](http://editorconfig.org/)for this?

Reply:
If it's possible, sure!

Reply:
Refs https://github.com/sonata-project/dev-kit/issues/353